client/blocks/login/two-factor-authentication/waiting-notification-approval.scss

```diff
@@ -1,3 +1,11 @@
+.two-factor-authentication__push-notification-approval {
+	width: 352px;
```

Comment:
Anoter reason for establishing a token, so we have a single value from somewhere global, for these widths. We want to avoid one-offs.

Reply:
It is removed in 431d9aa3c955265e4644210632f23889c26e8723

Reply:
Fixed in b7fafdcd9a61023d70a1ba3c80a90f1c0a3236e4